cmd/gpu_plugin/rm/gpu_plugin_resource_manager.go

```diff
@@ -79,14 +79,15 @@ type ResourceManager interface {
 	SetDevInfos(DeviceInfoMap)
 }
 
+//nolint: govet
 type resourceManager struct {
 	mutex            sync.RWMutex // for devTree updates during scan
 	deviceInfos      DeviceInfoMap
 	nodeName         string
 	clientset        kubernetes.Interface
+	prGetClientFunc  getClientFunc
```

Comment:
with `novet` added, do we need to change this struct at all?

Reply:
A-ha, type embedding is not used here. `mutex sync.RWMutex` is just a regular field. I think it'd be ok to just drop `//nolint` and realign the struct.

Reply:
ok, my bad. I asked whether it's needed here too...

Reply:
@mythi I thought that can still use 8 bytes less paddings.
@rojkov Ok, got it.! Let me drop it!